app/src/main/java/com/eveningoutpost/dexdrip/AddCalibration.java

```diff
@@ -146,28 +147,30 @@ public void onClick(final View v) {
                     final String string_value = value.getText().toString();
                     if (!TextUtils.isEmpty(string_value)) {
 
-                        //new Thread() {
-                        //    @Override
-                        //    public void run() {
-
                         try {
                             final double calValue = JoH.tolerantParseDouble(string_value);
 
-                            Calibration calibration = Calibration.create(calValue, getApplicationContext());
-                            if (calibration != null) {
-                                UndoRedo.addUndoCalibration(calibration.uuid);
-                                GcmActivity.pushCalibration(string_value, "0");
-
-                                final boolean wear_integration = Home.getPreferencesBoolean("wear_sync", false);//KS
-                                if (wear_integration) {
-                                    android.util.Log.d("AddCalibration", "start WatchUpdaterService with ACTION_SYNC_CALIBRATION");
-                                    startService(new Intent(v.getContext(), WatchUpdaterService.class).setAction(WatchUpdaterService.ACTION_SYNC_CALIBRATION));
+                            if(Home.get_master()) {
```

Comment:
Should this instead be `if (!Home.get_follower())` otherwise wont this break for a device which is not a master and not a follower (the default settings)